README.md

```diff
@@ -184,9 +158,43 @@ ebcli rest-server --trust-node
 An api collection for Postman (https://www.getpostman.com/) is provided [here](./cosmos-ethereum-bridge.postman_collection.json) which documents some API endpoints and can be used to interact with it.
 Note: For checking account details/balance, you will need to change the cosmos addresses in the URLs, params and body to match the addresses you generated that you want to check.
 
+## Running the relayer service
+
+For automated relaying, there is a relayer service that can be run that will automatically watch and relay events.
+
+```
+# Check ebrelayer connection to ebd
+ebrelayer status
+
+# Initialize the Relayer service for automatic claim processing
+ebrelayer init testing wss://ropsten.infura.io/ws 3de4ef81Ba6243A60B0a32d3BCeD4173b6EA02bb "LogLock(bytes32,address,bytes,address,uint256,uint256)" $(ebcli keys show validator -a)
+```
+
+The relayer will now watch the contract on Ropsten and create a claim whenever it detects a lock event.
+
+## Using the bridge
+
+With the application set up and the relayer running, you can now use Peggy by sending a lock transaction to the smart contract. You can do this from any Ethereum wallet/client that supports smart contract transactions.
+
+The easiest way to do this for now, assuming you have Metamask setup for Ropsten in the browser is to use remix or mycrypto as the frontend, for example:
+ - 1. Go to remix.ethereum.org
+ - 2. Compile Peggy.sol with solc v0.5.0
+ - 3. Set the environment as Injected Web3 Ropsten
+ - 4. On 'Run' tab, select Peggy and enter "0x3de4ef81Ba6243A60B0a32d3BCeD4173b6EA02bb" in 'At Address' field
+ - 5. Select 'At Address' to load the deployed contract
+ - 6. Enter the following for the variables under function lock():
+  _recipient = [HASHED_COSMOS_RECIPIENT_ADDRESS] *(for testuser cosmos1pjtgu0vau2m52nrykdpztrt887aykue0hq7dfh, enter "0x636f736d6f7331706a74677530766175326d35326e72796b64707a74727438383761796b756530687137646668")*
```

Comment:
Fine for functioning prototype, but will need to simplify this input for users somehow. We could either do the hash on contract or under the hood in a Peggy cli which allows users to send txs directly to the contract from within terminal.

Reply:
Yeh, this needs to be improved. Or we could do the hash on the relayer, couldn't we?